supportedBackends/google/pipelines/v2beta/src/main/scala/cromwell/backend/google/pipelines/v2beta/LifeSciencesFactory.scala

```diff
@@ -44,9 +37,6 @@ case class LifeSciencesFactory(applicationName: String, authMode: GoogleAuthMode
   with SSHAccessAction {
 
   override def build(initializer: HttpRequestInitializer): PipelinesApiRequestFactory = new PipelinesApiRequestFactory {
-    implicit lazy val googleProjectMetadataLabelDecoder: Decoder[ProjectLabels] = deriveDecoder
-
-    val ResourceManagerAuthScopes = List(CloudLifeSciencesScopes.CLOUD_PLATFORM)
```

Comment:
Curious what this SSH action has to do with moving around the VPC metadata reading?